css/style.css

```diff
@@ -48,7 +48,7 @@ a{
 }
 
 ul{
-  list-style: inside;
+  list-style: none;
```

Comment:
I presume this is the fix. I don't understand it but it works!

Reply:
I was actually kinda wondering that, too; my guess that it was partly setting the click handlers on a page load event; but if the change in the `ul` style is responsible, I have no idea how.

Reply:
@ndokos @dbutenhof I saw another design bug while fixing the accordion issue. The navbar was having list-style dots in them. So fixed them here. I thought of adding a comment but then felt adding a comment for only online in CSS might be inconsistent.

Reply:
Thanks, Anisha! That makes sense.